code/modules/mob/living/simple_animal/friendly/panda.dm

```diff
@@ -0,0 +1,20 @@
+/mob/living/simple_animal/pet/redpanda
```

Comment:
Move this pet addition to a new PR and hopefully get it merged fast instead of adding more burden to this one.

Reply:
This should be in a separate PR already, how did this get merged?

Reply:
Oh, they somehow got tangled up in the eigenstate fixes

Reply:
Fixed. Thanks!